govstack_api/apps.py

```diff
@@ -4,3 +4,7 @@
 class TestHarnessApiConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'govstack_api'
+
+    IM_CLIENT = 'eGovStack/GOV/90000009/digitalregistries'
```

Comment:
I think this could be loaded from .env instead of being hardcoded.

Reply:
done